core/unit_test/tools/include/ToolTestingUtilities.hpp

```diff
@@ -972,7 +972,8 @@ static uint64_t last_kernel_id;
 static uint32_t last_section_id;
 
 /** Subscribes to all of the requested callbacks */
-static void set_tool_events_impl(const ToolValidatorConfiguration& config) {
+static inline void set_tool_events_impl(
+    const ToolValidatorConfiguration& config) {
```

Comment:
Why?

Reply:
Many compilers started to complain about it, don't ask me why :laughing:

Reply:
We agreed to keep it. closing.

Reply:
What do the warnings look like?

Reply:
I had to got back in time, but here it is https://github.com/kokkos/kokkos/actions/runs/11017103540/job/30594206186 (and many other jobs had the same complain):
```bash
[  8%] Building CXX object core/unit_test/CMakeFiles/Kokkos_CoreTestCompileOnly.dir/TestStringManipulation.cpp.o
/Users/runner/work/kokkos/kokkos/tpls/gtest/../../core/unit_test/tools/include/ToolTestingUtilities.hpp:975:13: error: 'static' function 'set_tool_events_impl' declared in header file should be declared 'static inline' [-Werror,-Wunneeded-internal-declaration]
static void set_tool_events_impl(const ToolValidatorConfiguration& config) {
            ^
1 error generated.
make[2]: *** [core/unit_test/CMakeFiles/Kokkos_CoreUnitTest_Serial1.dir/serial/TestSerial_Graph.cpp.o] Error
```